src/main/java/org/commcare/formplayer/services/ResponseMetaDataTracker.java

```diff
@@ -12,11 +14,15 @@ public class ResponseMetaDataTracker {
 
     private boolean newInstall = false;
 
+    @Autowired
+    private FormplayerDatadog datadog;
+
     public boolean isAttemptRestore() {
         return attemptRestore;
     }
 
     public void setAttemptRestore(boolean attemptRestore) {
+        datadog.addRequestScopedTag("metadata", "attempt_restore");
```

Comment:
I don't think this needs a test like Codecov insists, but seems like it probably should use/add a tag as a constant for "metadata" here: https://github.com/dimagi/formplayer/blob/42cea06991728fa896198f31602a62e3850a70c2/src/main/java/org/commcare/formplayer/util/Constants.java#L124

Reply:
Good call [d2d609f](https://github.com/dimagi/formplayer/pull/1538/commits/d2d609f429396cde84cfae1a4740fb8aa0a79c31)

I'm also open to any tag name suggestion. I'm not sure if "metadata" is the best since all tags are in a sense metadata

Reply:
Is this usage necessarily/distinctly different from the `CATEGORY_TAG`? That's the only other place I would maybe see it fitting.